src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
-export interface Figure {
+// import { strict } from "assert";
 
+export interface Figure {
+  color: string;
+  shape: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape = 'triangle';
 
-}
+  constructor(
+    public color: string,
```

Comment:
create enum for colors

Reply:
but i get color as argument
how do i add it to him enum?

Reply:
enum should be type of color